pkg/util/util.go

```diff
@@ -18,23 +18,33 @@ package util
 
 import (
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"sort"
 	"strings"
 
 	"golang.org/x/mod/modfile"
 )
 
+const (
+	// Extension for go file.
+	ExtensionGo = ".go"
+)
+
 // CurrentPackage returns the go package of the current directory, or "" if it cannot
 // be derived from the GOPATH.
 // This logic is taken from k8.io/code-generator, but has a change of letting user pass the
 // directory whose pacakge is to be found.
 func CurrentPackage(dir string) (string, bool) {
-	goModPath, err := getGoModPath(dir)
+	absPath, err := filepath.Abs(dir)
+	if err != nil {
+		return "", false
+	}
+	goModPath, err := getGoModPath(absPath)
 	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+		return "", false
```

Comment:
Are we just returning with an empty string? Shouldn't we exit, because if we can't find the package path from go mod, then we would end up messing the input paths later while scaffolding.